lib/api/bgppeer.go

```diff
@@ -37,6 +39,10 @@ func (t BGPPeer) GetResourceMetadata() unversioned.ResourceMetadata {
 	return t.Metadata
 }
 
+func (t BGPPeer) String() string {
+	return fmt.Sprintf("BGPPeer(IP=%s, Scope=%s, Node=%s)", t.Metadata.PeerIP.IP.String(), t.Metadata.Scope, t.Metadata.Node)
```

Comment:
Not too fussed about this, but perhaps if scope is node and node=="" we could omit node from the name.